README.md

```diff
@@ -407,62 +412,303 @@ $ tkn resource ls
+```
+
+* Run following command to apply Triggertemplate.
+
+```bash
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/02_template.yaml
+```
+
+
+####  Trigger Binding
+TriggerBindings is a map enable you to capture fields from an event and store them as parameters, and replace them in triggerTemplate whenever an event occurs.
+
+The definition of our TriggerBinding is given in `03-triggers/01_binding.yaml`.
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: vote-app-pipelinebinding
+spec:
+  params:
+  - name: git-revision
+    value: $(body.head_commit.id)
+  - name: namespace
+    value: pipelines-tutorial
+  - name: git-repo-url
+    value: $(body.repository.url)
+  - name: git-repo-name
+    value: $(body.repository.name)
+```
+The exact paths (keys) of parameter we need can be found by examining the event payload (eg: GitHub events).
+
+
+Run following command to apply Triggertemplate.
+
+```bash
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/01_binding.yaml
+```
+>**Note**:- The parameter namespace is hardcoded as `pipelines-tutorial`. It can be also parameterized.
+
+#### Event Listener
+
+This component sets up a Service and listens for events. It also connects a TriggerTemplate to a TriggerBinding, into an
+  [addressable](https://github.com/knative/eventing/blob/master/docs/spec/interfaces.md)
+  endpoint (the event sink)
+
+The definition for our EventListener can be found in
+`03-triggers/03_event_listener.yaml`.
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: voteapp-listener
+spec:
+  serviceAccountName: pipeline
+  triggers:
+  - name: vote-app-trig
+    bindings:
+    - name: vote-app-pipelinebinding
+    template:
+      name: vote-app-triggertemplate
+```
+* Run following command to create Triggertemplate.
+
+```bash
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/03_event_listener.yaml
+```
+
+>***Note***: EventListener will setup a Service. We need to expose that Service as an OpenShift Route to make it publicly accessible.
+
+* Run below command to expose eventlistener service as a route
+
+```bash
+$ oc expose svc el-voteapp-listener
+```
+
+
+
+
+## Configuring GitHub WebHooks
+
+Now we need to configure webhook-url on [backend](https://github.com/openshift-pipelines/vote-api) and [frontend](https://github.com/openshift-pipelines/vote-ui) source code repositories with the Route we exposed in the previously.
+
+* Run below command to get webhook-url
+
+```bash
+$ echo "URL: $(oc  get route el-voteapp-listener --template='http://{{.spec.host}}')"
+```
+
+>***Note:***
+>
+>Fork the [backend](https://github.com/openshift-pipelines/vote-api) and [frontend](https://github.com/openshift->pipelines/vote-ui) source code repositories so that you have sufficient privileges to configure GitHub webhooks.
```

Comment:
change 
```suggestion
>Fork the [backend](https://github.com/openshift-pipelines/vote-api) and [frontend](https://github.com/openshift-pipelines/vote-ui) source code repositories so that you have sufficient privileges to configure GitHub webhooks.
```